### PR TITLE
Clarify details about versioning and limitations

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
@nachoalonsoportillo Thanks for contacting us with feedback about our product documentation. Depending on the complexity, it could take from a few business days to a few weeks to look further into your question, issue, or suggestion. We'll contact you if we need further information, and you'll receive an email message each time we "@mention" you in a comment. 

Thanks again for submitting your feedback. Your suggestions help improve our documentation.

File: data-explorer/kusto/management/undo-drop-table-command.md [hunk 1]
```diff
@@ -7,13 +7,15 @@ ms.date: 02/04/2020
 ---
 # .undo drop table
 
-The `.undo` `drop` `table` command reverts a drop table operation to a specific database version.
+The `.undo drop table` command reverts a drop table operation to a specific database version.
+
+That database version is the one before the drop operation was executed, and corresponds to the highest minor version of one major version before the one assigned by the DROP-TABLE event written in the journal when the table was dropped. More details on how to get such version can be found in the **How to find the required database version** section below.
```
Comment (v-dkorn) [inline data-explorer/kusto/management/undo-drop-table-command.md:12]:
```suggestion
The `.undo drop table` command reverts a drop table operation to a specific database version. The database version must be the version prior to the table deletion. To find the correct version to use with the command, follow the instructions in [How to find the required database version](#how-to-find-the-required-datatbase-version).
```

**Comment by v-dkorn** [comment]
@nachoalonsoportillo We realized additional changes that needed to be made to this file due to your contribution. Your suggestions will be incorporated into the changes and published soon. Thanks again!

**Comment by v-dkorn** [comment]
#please-close

**Pull request closed without merging**